// // 3 - Reescreva o teste do exerc√≠cio anterior, desta vez utilizando a sintaxe de async/await .
// Dica: Utilize o try/catch para o caso de erro.